src/v3.rs

```diff
@@ -76,6 +81,25 @@ pub struct Personalization {
     send_at: Option<u64>,
 }
 
+/// An attachment block for a V3 message. Content and filename are required. If the
+/// mime_type is unspecified, the email will use Sendgrids default for attachments
```

Comment:
Add an apostrophe to "Sendgrid's" and after "attachments" add "which is" so it flows a little better.